x-pack/plugins/observability/public/components/shared/slo/slo_selector/slo_selector.stories.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { SloSelector as Component } from './slo_selector';
+
+export default {
+  component: Component,
+  title: 'app/SLOs/Shared/SloSelector',
```

Comment:
💬 Need to find a way to mock the hook to fake some data. Any ideas?

Reply:
Found https://storybook.js.org/addons/storybook-addon-manual-mocks. Every component that's rendered in Storybook will check for a `__mock__` folder when doing imports and if that's available, use that instead. 

Seems to work!
<img width="507" alt="Screenshot 2022-12-06 at 12 56 33" src="https://user-images.githubusercontent.com/535564/205905459-93262494-b0f6-4ecd-bc05-c6cf9253269e.png">

You might want to do a bit of clicking around in other stories to see if this doesn't lead to issues.

Reply:
Fixed! Removed the plugin and used the same logic inside our own SB config, which allows to use a different folder (`__storybook_mocks__`) instead of `__mocks__` .